### PR TITLE
[PR] Support multi pose estimation on LiveImageViewController

### PR DESCRIPTION
**Pull request by tucan9389**
## PR Points

- Support OpenPose on LiveImageViewController
- Can select visible part on LiveImageViewController with `UIButton`s

## Demo

- source video: https://youtu.be/qjc23BuYQh8

### CODE REVIEW

File: PoseEstimation-TFLiteSwift/LiveImageViewController.swift [hunk 1]
```diff
@@ -13,16 +13,34 @@ class LiveImageViewController: UIViewController {
...
+        var partIndexes: [String: Int] = [:]
+        poseEstimator.partNames.enumerated().forEach { offset, partName in
+            partIndexes[partName] = offset
+        }
+        return partIndexes
+    }()
+    var selectedPartName: String = "ALL"
+    var selectedPartIndex: Int? {
+        guard let partName = selectedPartName.components(separatedBy: "(").first else { return nil }
+        return partIndexes[partName]
```
Comment (MBKwon) [inline PoseEstimation-TFLiteSwift/LiveImageViewController.swift:33]: Indice is a plural of index. indexes is not a plural.

Comment (tucan9389) [reply PoseEstimation-TFLiteSwift/LiveImageViewController.swift:33]: @MBKwon

Here is similar discuss https://github.com/tucan9389/PoseEstimation-TFLiteSwift/pull/41#discussion_r417717831 

Apple's official document and API use the `indices`, but `indexes` have been also used ([link](https://developer.apple.com/documentation/swift/collection/1641719-indices)). So I think I can also use the `indexes` name. Do you have any opinion about it?

Comment (MBKwon) [reply PoseEstimation-TFLiteSwift/LiveImageViewController.swift:33]: I can’t find ‘indexes’ in the link you provide.

Comment (tucan9389) [reply PoseEstimation-TFLiteSwift/LiveImageViewController.swift:33]: You can see the discussion in the following link.
- https://github.com/tucan9389/PoseEstimation-TFLiteSwift/pull/41#discussion_r417716157

Here is the screenshot.

image

If you cannot find it with the above link, please tell me again.

Thank you.

Comment (MBKwon) [reply PoseEstimation-TFLiteSwift/LiveImageViewController.swift:33]: I found about it. You can use both of them. But they have some differences. You choose one you want to use after you see the below. 


https://www.nasdaq.com/articles/indexes-or-indices-whats-the-deal-2016-05-12

Comment (tucan9389) [reply PoseEstimation-TFLiteSwift/LiveImageViewController.swift:47]: When I googled, there were two terms of meaning. `Indices` have been used in mathematical(Figure 1), and `Indexes` have been used in publishing(Figure 2). In this context, `Indexes` is more fit but I think there is no precise answer about the subject[[3]](https://stackoverflow.com/a/1379069/4160632). 

So I'll use the `indexes` in this repository. If you have any other opinions, please comment feel free.

| Figure 1. Index of mathematic[[1]](https://images.app.goo.gl/ot6ytTzGaKxsg16x8) |
| ---- |
|  |

| Figure 2. Index of publishing[[2]](https://images.app.goo.gl/a4owkwwa1yQ6gWXA6) |
| ---- | 
|  |

[1] https://images.app.goo.gl/ot6ytTzGaKxsg16x8
[2] https://images.app.goo.gl/a4owkwwa1yQ6gWXA6
[3] https://stackoverflow.com/a/1379069/4160632

**Pull request merged**